web/app/(commonLayout)/app/(appDetailLayout)/[appId]/layout.tsx

```diff
@@ -41,12 +42,14 @@ const AppDetailLayout: FC<IAppDetailLayoutProps> = (props) => {
   const pathname = usePathname()
   const media = useBreakpoints()
   const isMobile = media === MediaType.mobile
-  const { isCurrentWorkspaceEditor } = useAppContext()
+  const { isCurrentWorkspaceEditor, isLoadingCurrentWorkspace } = useAppContext()
   const { appDetail, setAppDetail, setAppSiderbarExpand } = useStore(useShallow(state => ({
     appDetail: state.appDetail,
     setAppDetail: state.setAppDetail,
     setAppSiderbarExpand: state.setAppSiderbarExpand,
   })))
+  const [isLoadingAppDetail, setIsLoadingAppDetail] = useState(false)
+  const [appDetailRes, setAppDetailRes] = useState<App | null>(null)
```

Comment:
It seems like the variable is not being used. Would it be safe to remove it ?

Reply:
appDetailRes is used in this section.
https://github.com/barabicu/dify/blob/40b54e5d844ab6c615cbbad8f9c3f3c7a82a1e39/web/app/(commonLayout)/app/(appDetailLayout)/%5BappId%5D/layout.tsx#L125-L128